crates/llm-base/src/loader.rs

```diff
@@ -741,6 +752,15 @@ impl<'a> FileContext<'a> {
             }
         }
 
+        // The tensor name is truncated to 32 characters.
+
+        let tensor_name = if name.len() > 32 {
```

Comment:
32 or 48?

Reply:
Was 32, now is 48. I will use `GGML_MAX_NAME`.